resolwe_bio/utils/test.py

```diff
@@ -93,6 +99,19 @@ def prepare_expression(self, f_rc='exp_1_rc.tab.gz',
         return self.run_process('upload-expression', inputs)
 
 
+class BioProcessTestCase(TransactionBioProcessTestCase, DjangoTestCase):
```

Comment:
I guess we want as many tests to be *vanilla*, i.e. do everything in a transaction that is then reverted.

But I'm open to arguments for using just the non-transaction one.